CONTRIBUTING.md

```diff
@@ -6,8 +6,7 @@ These guidelines are designed to make it as easy as possible to get involved. If
 
 Before you start you'll need to set up a free [GitHub][link_github] account and sign in. Here are some [instructions][link_signupinstructions].
 If you are not familiar with version control systems such as git,
- [introductions and tutorials](http://www.reproducibleimaging.org/module-reproducible-basics/02-vcs/)
- may be found on [ReproducibleImaging.org](https://www.reproducibleimaging.org/).
+ introductions and tutorials may be found [here](http://www.reproducibleimaging.org/module-reproducible-basics/02-vcs/).
```

Comment:
W3C has [recommendations](https://www.w3.org/QA/Tips/noClickHere) for link text. I think we can rewrite this a bit better (and fix the broken link):

```suggestion
we recommend the [VCS module](http://www.reproducibleimaging.org/module-reproducible-basics/02-vcs/)
available from [ReproNim](http://www.reproducibleimaging.org/).
```